### PR TITLE
sql: add `EXPLAIN ANALYZE` output format

### PR DESCRIPTION
**Pull request by CaitinChen**
@lilin90 @lysu PTAL
Via https://github.com/pingcap/docs/pull/715

Since we do not add the Bikeshare example database to docs-cn currently, this PR only includes the descriptive section without the example.

### CODE REVIEW

File: sql/understanding-the-query-execution-plan.md [hunk 1]
```diff
@@ -28,6 +28,14 @@ TiDB 优化器会根据当前数据表的实际情况来选择最优的执行计
 | task          | 当前这个 operator 属于什么 task。目前的执行计划分成为两种 task，一种叫 **root** task，在 tidb-server 上执行，一种叫 **cop** task，并行的在 TiKV 上执行。当前的执行计划在 task 级别的拓扑关系是一个 root task 后面可以跟许多 cop task，root task 使用 cop task 的输出结果作为输入。cop task 中执行的也即是 TiDB 下推到 TiKV 上的任务，每个 cop task 分散在 TiKV 集群中，由多个进程共同执行。 |
 | operator info | 每个 operator 的详细信息。各个 operator 的 operator info 各有不同，详见 [Operator Info](#operator-info)。                   |
 
+## <span id="explain-analyze-output-format">`EXPLAIN ANALYZE` 输出格式</span>
+
+作为 `EXPLAIN` 语句的延伸，`EXPLAIN ANALYZE` 语句执行查询并提供 `execution info` 列中额外的执行统计信息：
```
Comment (lilin90) [inline sql/understanding-the-query-execution-plan.md]:
```suggestion
作为 `EXPLAIN` 语句的扩展，`EXPLAIN ANALYZE` 语句执行查询并在 `execution info` 列中提供额外的执行统计信息。具体如下：
```

File: sql/understanding-the-query-execution-plan.md [hunk 2]
```diff
@@ -28,6 +28,14 @@ TiDB 优化器会根据当前数据表的实际情况来选择最优的执行计
...
+
+* `time` 显示从进入执行器到离开执行器的全部 wall time，包括所有子执行器操作的全部执行时间。如果该执行器从父执行器中多次调出 (`loops`)，这个时间就是累积的时间。
+* `loops` 是执行器从父执行器中调出的次数。
+* `rows` 是执行器返回的行的总数。例如，可以将 `count` 列的精度和 `execution_info` 列中的 `rows`/`loops` 值进行对比，以此评价 query optimizer 的估算的精确度。
```
Comment (lilin90) [inline sql/understanding-the-query-execution-plan.md]:
```suggestion
* `rows` 是执行器返回的行的总数。例如，可以将 `count` 列的精度和 `execution_info` 列中的 `rows`/`loops` 值进行对比，据此评定查询优化器估算的精确度。
```

**Comment by CaitinChen** [comment]
@lilin90 Thanks. PTAL again. @lysu PTAL.

**Comment by CaitinChen** [comment]
@lysu Thanks. PTAL

File: sql/understanding-the-query-execution-plan.md [hunk 3]
```diff
@@ -28,6 +28,14 @@ TiDB 优化器会根据当前数据表的实际情况来选择最优的执行计
 | task          | 当前这个 operator 属于什么 task。目前的执行计划分成为两种 task，一种叫 **root** task，在 tidb-server 上执行，一种叫 **cop** task，并行的在 TiKV 上执行。当前的执行计划在 task 级别的拓扑关系是一个 root task 后面可以跟许多 cop task，root task 使用 cop task 的输出结果作为输入。cop task 中执行的也即是 TiDB 下推到 TiKV 上的任务，每个 cop task 分散在 TiKV 集群中，由多个进程共同执行。 |
 | operator info | 每个 operator 的详细信息。各个 operator 的 operator info 各有不同，详见 [Operator Info](#operator-info)。                   |
 
+## <span id="explain-analyze-output-format">`EXPLAIN ANALYZE` 输出格式</span>
+
+作为 `EXPLAIN` 语句的扩展，`EXPLAIN ANALYZE` 语句执行查询并在 `execution info` 列中提供额外的执行统计信息。具体如下：
+
+* `time` 显示从进入 operator 到离开 operator 的全部 wall time，包括所有子 operator 操作的全部执行时间。如果该 operator 从父 operator 中多次调出 (`loops`)，这个时间就是累积的时间。
+* `loops` 是当前 operator 被父 operator 的调用次数。
+* `rows` 是 operator 返回的行的总数。例如，可以将 `count` 列的精度和 `execution_info` 列中的 `rows`/`loops` 值进行对比，据此评定查询优化器估算的精确度。
```
[the remark this answers is not in the document]
Comment (CaitinChen) [reply sql/understanding-the-query-execution-plan.md]: @lysu I want to keep my wording.
- Generally, we do not use "I"/"we" in technical docs.
- In the English doc, "for example" is used, so I translate it into “例如” in Chinese.
- Your comma after “所以” is in English. @_@

[the remark this answers is not in the document]
Comment (lysu) [reply sql/understanding-the-query-execution-plan.md]: @CaitinChen  ok :laughing:

**Comment by lysu** [comment]
LGTM

**Comment by CaitinChen** [comment]
Please sign CLA. ^_^

**Comment by lysu** [comment]
@CaitinChen signed~

**Comment by lilin90** [comment]
@CaitinChen I'm curious that why the "executor" in the English version is replaced with "operator" in the Chinese version. :)

**Comment by CaitinChen** [comment]
@lysu @lilin90

**Pull request merged**